docs/sources/operators-guide/configure/reference-configuration-parameters/index.md

```diff
@@ -3015,19 +3015,13 @@ tsdb:
   # CLI flag: -blocks-storage.tsdb.memory-snapshot-on-shutdown
   [memory_snapshot_on_shutdown: <boolean> | default = false]
 
-  # (experimental) The size of the write queue used by the head chunks mapper.
-  # Lower values reduce memory utilisation at the cost of potentially higher
-  # ingest latency. Value of 0 switches chunks mapper to implementation without
-  # a queue. This flag is only used if the new chunk disk mapper is enabled with
-  # -blocks-storage.tsdb.new-chunk-disk-mapper.
+  # (advanced) The size of the write queue used by the head chunks mapper. Lower
+  # values reduce memory utilisation at the cost of potentially higher ingest
+  # latency. Value of 0 switches chunks mapper to implementation without a
+  # queue.
   # CLI flag: -blocks-storage.tsdb.head-chunks-write-queue-size
   [head_chunks_write_queue_size: <int> | default = 0]
```

Comment:
I would change the default to 1M. It's what we use at Grafana Labs in prod, and we should upstream for the rest of the world too.

Reply:
Done in the latest commit https://github.com/grafana/mimir/pull/2762/commits/c96cca3444be67985a7f4fca90c9e7ec94859bc2